sites/fast-website/src/app/css/style.css

```diff
@@ -327,15 +332,14 @@ fast-badge::part(control) {
   site-navigation .anchor-wrapper {
     display: flex;
     flex-direction: column;
-    position: absolute;
+    position: fixed;
```

Comment:
fixed-position elements nested in fixed-position elements are redundant

Reply:
This has been resolved per your help through discord :)